pkg/glusterutils/local_peerID.go

```diff
@@ -0,0 +1,70 @@
+package glusterutils
+
+import (
+	"bufio"
+	"errors"
+	"flag"
+	"os"
+	"regexp"
+	"strings"
+)
+
+var (
+	defaultGlusterd1Workdir = ""
+	defaultGlusterd2Workdir = ""
+)
+
+var (
+	glusterMgmt     = flag.String("gluster-mgmt", "glusterd1", "Choice of GlusterD version i.e glusterd1 or glusterd2, Default is glusterd1")
```

Comment:
Do not use flags inside utility file. Move all flags to main function which handles the user arguments

Reply:
Ack